docs/make.jl

```diff
@@ -1,5 +1,9 @@
 Base.HOME_PROJECT[] = abspath(Base.HOME_PROJECT[]) # JuliaLang/julia/pull/28625
 
+push!(LOAD_PATH, joinpath(@__DIR__, ".."))               # add CLIMA env
```

Comment:
@simonbyrne are you saying that we should just remove this line since we already depend on CLIMA?

Reply:
I removed this, is that what we should do?